docker-compose.yml

```diff
@@ -158,25 +158,29 @@ services:
     depends_on:
       <<: [ *svc-depends-on ]
 
-  svc-bie-kafka:
-    # Expose healthcheck port for end2end test
-    ports:
-      - "10301:10301"
-    profiles: [ "all","svc","kafka" ]
-    image: va/abd_vro-svc-bie-kafka:latest
-    <<: [ *common-sde-security, *common-security-opt ]
-    environment:
-      <<: [*rabbitmq-placeholder-vars, *postgres-vars, *common-vars]
-      BIE_KAFKA_PLACEHOLDERS_BROKERS: mock-bie-kafka:9092
-      BIE_KAFKA_PLACEHOLDERS_SCHEMA_REGISTRY: mock-bie-schema-registry:8081
-      BIE_KAFKA_KEYSTORE_INBASE64: ${BIE_KAFKA_KEYSTORE_INBASE64}
-      BIE_KAFKA_KEYSTORE_PASSWORD: ${BIE_KAFKA_KEYSTORE_PASSWORD}
-      BIE_KAFKA_TRUSTSTORE_INBASE64: ${BIE_KAFKA_TRUSTSTORE_INBASE64}
-      BIE_KAFKA_TRUSTSTORE_PASSWORD: ${BIE_KAFKA_TRUSTSTORE_PASSWORD}
-    networks:
-      - intranet
-    depends_on:
-      <<: [ *svc-depends-on ]
+  # NOTE: svc-bie-kafka workflows are tied to inactive services and commented out to prevent unnecessary execution and updates.
+  # Documentation:
+  # - Disabling issue: https://github.com/department-of-veterans-affairs/abd-vro/issues/3802
+  # - BIE Docs: https://github.com/department-of-veterans-affairs/abd-vro/issues/3753
+  # svc-bie-kafka:
+  #   # Expose healthcheck port for end2end test
+  #   ports:
+  #     - "10301:10301"
+  #   profiles: [ "all","svc","kafka" ]
+  #   image: va/abd_vro-svc-bie-kafka:latest
+  #   <<: [ *common-sde-security, *common-security-opt ]
+  #   environment:
+  #     <<: [*rabbitmq-placeholder-vars, *postgres-vars, *common-vars]
+  #     BIE_KAFKA_PLACEHOLDERS_BROKERS: mock-bie-kafka:9092
+  #     BIE_KAFKA_PLACEHOLDERS_SCHEMA_REGISTRY: mock-bie-schema-registry:8081
+  #     BIE_KAFKA_KEYSTORE_INBASE64: ${BIE_KAFKA_KEYSTORE_INBASE64}
+  #     BIE_KAFKA_KEYSTORE_PASSWORD: ${BIE_KAFKA_KEYSTORE_PASSWORD}
+  #     BIE_KAFKA_TRUSTSTORE_INBASE64: ${BIE_KAFKA_TRUSTSTORE_INBASE64}
+  #     BIE_KAFKA_TRUSTSTORE_PASSWORD: ${BIE_KAFKA_TRUSTSTORE_PASSWORD}
+  #   networks:
+  #     - intranet
+  #   depends_on:
+  #     <<: [ *svc-depends-on ]
 
   vro-streamlit:
```

Comment:
We could probably comment out streamlit things too, any thoughts @dfitchett?

Reply:
We might as well. I'll push up a commit after my latest commit goes fully green 😄

Reply:
Yeah, but i think that can probably happen separately! Lets keep our scope narrowed to this service.

Reply:
Woops, I pushed up a commit doing it:
https://github.com/department-of-veterans-affairs/abd-vro/pull/3814/commits/97a81b399365a6c4f83db0dcb23a6e5f3d6c1ae1

Happy to split it off, or just merge it.

Reply:
Nah no worries!